gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/components/AltTextSection.kt

```diff
@@ -0,0 +1,174 @@
+package com.gravatar.quickeditor.ui.components
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.material3.LocalTextStyle
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import com.gravatar.quickeditor.R
+import com.gravatar.quickeditor.ui.avatarpicker.MainContentSection
+import com.gravatar.restapi.models.Avatar
+import com.gravatar.ui.GravatarTheme
+import java.net.URI
+import java.net.URL
+
+@Composable
+internal fun AltTextSection(
+    state: MainContentSection.AltTextUiState,
+    onAltTextChanged: (String) -> Unit,
+    onWhatIsAltTextClicked: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    val sectionPadding = 16.dp
+    var altText by remember { mutableStateOf(state.avatar.altText) }
```

Comment:
⚠️ We should at least use `rememberSavable` here, I've lost the text on the screen orientation change.

Reply:
Ideally, we should have this in the viewModel, so we can use this for any logic that depends on it (max limit for example).

Reply:
Everything is in the viewmodel now.